pkg/kubelet/kubeletconfig/configsync.go

```diff
@@ -198,7 +198,7 @@ func restartForNewConfig(eventClient v1core.EventsGetter, nodeName string, sourc
 	// we directly log and send the event, instead of using the event recorder,
 	// because the event recorder won't flush its queue before we exit (we'd lose the event)
 	event := makeEvent(nodeName, apiv1.EventTypeNormal, KubeletConfigChangedEventReason, message)
-	klog.V(3).Infof("Event(%#v): type: '%v' reason: '%v' %v", event.InvolvedObject, event.Type, event.Reason, event.Message)
+	klog.V(3).InfoS("Event created", "event", klog.KObj(event))
```

Comment:
`klog.KObj` is used to reference an object, but it doesn't dump object fields like in this case.

I would propose to add other fields separatly `eventType`, `eventReason`, `eventMessage`, `eventInvolvedObject`, similar to how we pass `podUID` as a separate field. In future we may expand `klog.KObj to put more data about the object, but for now let's have them listed explicitly.